stable/stretch-perl/Dockerfile

```
@@ -90,7 +90,8 @@ RUN set -x \
 	fi
 
 # implement changes required to run NGINX as an unprivileged user
-RUN chown -R :0 /var/cache/nginx \
+RUN usermod -u 1001 nginx \
+	chown -R 1001:0 /var/cache/nginx \
```

Also missing the leading `&&`?

```suggestion
	&& chown -R 1001:0 /var/cache/nginx \
```